app/templates/people/person/contacts/index.hbs

```diff
@@ -0,0 +1,73 @@
+<div class="au-o-box">
+  <AuToolbar>
+    <AuToolbarGroup>
+      <AuHeading>
+        Contactgegevens
+      </AuHeading>
+    </AuToolbarGroup>
+    <AuToolbarGroup>
+      {{! TODO: Replace this with the "new" route once it is created }}
+      <AuLink
```

Comment:
I know it's like this in the design, but I wonder what page this will show? I assumed the contact information was related to the position?

Reply:
Yeah, it isn't very clear. Maybe it's better to ask at Figma?